src/s3/util/fileViewerManager.ts

```diff
@@ -135,65 +138,46 @@ export class S3FileViewerManager {
         const targetLocation = vscode.Uri.file(targetPath)
 
         if (this.cacheArns.has(fileNode.file.arn)) {
-            //get it from temp IF it hasn't been recently modified, then return that
-
-            getLogger().debug(`cache is working!, found ${fileNode.file.key} in cache`)
-            showOutputMessage(`cache is working!, found ${fileNode.file.key} in cache`, this.outputChannel) //TODOD:: debug log remove
+            getLogger().info(`FileViewer: found file ${fileNode.file.key} in cache`)
 
-            //explorer (or at least the S3Node) needs to be refreshed to get the last modified date from S3
+            //Explorer (or at least the S3Node) needs to be refreshed to get the last modified date from S3
             const newNode = await this.refreshNode(fileNode)
             if (!newNode) {
-                showOutputMessage(
-                    `!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! something happeneded`,
-                    this.outputChannel
-                ) //TODOD:: debug log, remove
+                getLogger().error(`FileViewer: Error, refreshNode() returned undefined with file: ${fileNode.file.key}`)
+                getLogger().debug(`Cache contains: ${this.cacheArns}`)
```

Comment:
I like the idea of displaying the cache in a debug message, but this probably shouldn't be reliant on newNode being undefined and causing an error. Probably better to output this when you're interacting directly with the cache, e.g. adding, removing, checking.